frontend/pages/hosts/details/cards/Packs/PackTable/PackTableConfig.tsx

```diff
@@ -1,157 +0,0 @@
-import React from "react";
```

Comment:
Note: These changes are removing a legacy table "Packs" that is available for super old users that might be still be secretly using packs.

I can't find a design dev note in Figma or conversation in the issue ticket that confirms this change.

cc: @rachaelshaw / @noahtalerman